React/Week1/my-app-week1-task/app/page.js

```diff
@@ -0,0 +1,32 @@
+"use client"
+
+
+import { GreetingComponent } from "@/components/Greeting-component/GreetingComponent";
+import { CardComponent } from "@/components/Card-component/CardComponent";
+import { CardsComponent } from "@/components/Cards-component/CardsComponent";
+import { Person } from "@/components/Person-component/PersonComponent";
+import { Button } from "@/components/Button-component/Button";
+import { DangerButton } from "@/components/DangerButton-component/DangerButton";
+import { TextInput } from "@/components/TextInput-component/TextInput";
+import { ProfileImage } from "@/components/ProfileImage-Component/ProfileImage";
+import { Header } from "@/components/Header/header"; 
+import { Footer } from "@/components/Footer/Footer";
+
+
+export default function Home() {
+  return (
+    <div>
+      <GreetingComponent/>
+      <CardComponent/>
+      <CardsComponent/>
+      <Person/>
+      <Button appearance="btn-primary" text="Click me" action={() => console.log("click")} />
+      <DangerButton appearance="btn-secondary" text="Don't click me" action={() => console.log("click")} />
+      <TextInput appearance="not yet" placeholder="Type something here" action={()=> console.log("submit")}/>
```

Comment:
here we have a mis-connection between how the props are declared in the component file, and what we are passing when the component is imported.

in TextInput.jsx you declare 4 props: 'appearance', 'placeholder', 'type' and 'value'

when importing the component here you should pass arguments for the same 4 but instead 'type' and 'value' are missing, and instead we have 'action' that was not declared!

these kinds of mis-matches are going to cause for components not to work correctly or even break completely and throw compiler/runtime errors. please be very careful and consistent with these!